haas/cli.py

```diff
@@ -175,16 +205,16 @@ def list_projects():
     do_get(url)
 
 @cmd
-def project_add_user(project, user):
+def user_add_project(user, project):
```

Comment:
Interesting - changing this would represent a departure between the CLI and API schemas, though in principle that is ok since the CLI is meant to be procedural and user friendly whereas the API has different design goals.

We have #498 to create a proper client library.

How strongly do you feel about this? I prefer `project_*` since it will be easier for users to find the CLI but won't hold up the PR for it.